backend/python/start_all_python_services.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+echo "Starting all Python backend services..."
+
+# Activate virtual environment
+source venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)